go.mod

```diff
@@ -73,6 +74,7 @@ require (
 go 1.16
 
 replace (
+	github.com/cs3org/go-cs3apis => github.com/ishank011/go-cs3apis v0.0.0-20210806135412-33c0570675bf
```

Comment:
Looks all right to me, and with the latest patch in the WOPI server we demonstrated it works.
Just do not forget to revert this before merging the PR.